emissionsapi/db.py

```diff
@@ -219,28 +230,50 @@ def get_statistics(session, polygon=None, begin=None, end=None,
         sqlalchemy.func.min(Carbonmonoxide.timestamp),
         sqlalchemy.func.max(Carbonmonoxide.timestamp),
         interval).group_by(interval)
-    return filter_query(query, polygon=polygon, begin=begin, end=end)
+    return filter_query(
+        query, polygon=polygon, point=point, begin=begin, end=end)
 
 
-def filter_query(query, polygon=None, begin=None, end=None):
+def filter_query(
+        query, polygon=None, point=None, radius=0.2, begin=None, end=None):
```

Comment:
I will check if there are any differences in speed between `ST_DWITHIN` and `ST_WITHIN`

Reply:
Only now do I realize that two different functions are being called. Sorry.
But you could still do:

```py
if wkt:
    # Filter with geometric form
    if radius:
        query = query.filter(geoalchemy2.func.ST_DWITHIN(
            Carbonmonoxide.geom,
            wkt,
            radius))
    else:
        query = query.filter(geoalchemy2.func.ST_WITHIN(
            Carbonmonoxide.geom,
            wkt))
```

Reply:
Seems like the results differ anyway sometimes. Although I currently do not know what the real difference is. But never mind. That is not the point here.

I would like to go with `distance` instead of `radius`. But I still like the idea.